url.bs

```diff
@@ -1016,10 +1352,11 @@ actually doing that with the editors of this document first.
          <li><p>If <a>c</a> is a U+002E (.) and <var>numbersSeen</var> is less than 4, then increase
          <var>pointer</var> by 1.
 
-         <li>Otherwise, <a>validation error</a>, return failure.
+         <li>Otherwise, <a>IPv4-in-IPv6-too-many-parts</a> <a>validation error</a>, return failure.
```

Comment:
This can also happen when c is not `.`. E.g., `https://[ffff::12xyz]`. Should we perhaps reuse IPv4-in-IPv6-unexpected-code-point here and just call out all the different cases in examples?

Reply:
`https://[ffff::12xyz]`[(demo)](https://url-demo.trowbotham.com/?url=aAB0AHQAcABzADoALwAvAFsAZgBmAGYAZgA6ADoAMQAyAHgAeQB6AF0A) reports IPv6-unexpected-delimiter for me, but modifying it slightly to `https://[ffff::12.12xyz]`[(demo)](https://url-demo.trowbotham.com/?url=aAB0AHQAcABzADoALwAvAFsAZgBmAGYAZgA6ADoAMQAyAC4AMQAyAHgAeQB6AF0A) shows the problem as the parser needs to encounter at least one `.` before taking that code path.

Your suggestion of using IPv4-in-IPv6-unexpected-code-point seems reasonable to me. I'd be interested to hear other peoples opinions.